README.md

```diff
@@ -230,15 +230,15 @@ showLocation({
   cancelText: 'This is the cancel button text', // optional (default: 'Cancel')
   appsWhiteList: ['google-maps'], // optionally you can set which apps to show (default: will show all supported apps installed on device)
   naverCallerName: 'com.example.myapp', // to link into Naver Map You should provide your appname which is the bundle ID in iOS and applicationId in android.
-  // appTitles: { 'google-maps': 'My custom Google Maps title' }, // optionally you can override default app titles
-  // app: 'uber',  // optionally specify specific app to use
+  appTitles: { 'google-maps': 'My custom Google Maps title' }, // optionally you can override default app titles
+  app: 'uber',  // optionally specify specific app to use
```

Comment:
I think the idea here was the disable these by default, in case someone just copies the code snippet without looking and is confused about the fact that a specific app is opened rather than showing the list of options.

But maybe that means we should have a short simple example and then one that shows all the available options?

Reply:
Oops sorry I ignored your reply before.

We already have "optionally" comments after these options. Other optional options are not commented, so IMO it could be better to keep these 2 options consistent, i.e. uncommented. 

<img width="715" alt="image" src="https://github.com/includable/react-native-map-link/assets/18205362/0731ff69-4a34-4c52-b0fb-ee348370c022">